storage/pipeline/commit_batch.go

```diff
@@ -234,7 +235,11 @@ func (b *CommitBatcher) maybeStartBatch(notif bool) ([]sealiface.CommitBatchRes,
 	if individual {
 		res, err = b.processIndividually(cfg)
 	} else {
-		res, err = b.processBatch(cfg)
+		var sectors []abi.SectorNumber
+		for sn := range b.todo {
+			sectors = append(sectors, sn)
+		}
+		res, err = b.processBatch(cfg, sectors)
```

Comment:
This probably needs to be a loop to actually retry

Reply:
As mentioned above, i'm not actually adding retries. Just splitting the batch recursively until it gets to an appropriate size without first sending the msg